src/build.proj

```diff
@@ -10,7 +10,6 @@
     <ProjectReference Condition="'$(BuildManagedTools)' == 'true'" Include="tools/runincontext/runincontext.csproj" />
     <ProjectReference Condition="'$(BuildManagedTools)' == 'true'" Include="tools/r2rdump/R2RDump.csproj" />
     <ProjectReference Condition="'$(BuildManagedTools)' == 'true'" Include="tools/ReadyToRun.SuperIlc/ReadyToRun.SuperIlc.csproj" />
-    <ProjectReference Condition="'$(BuildManagedTools)' == 'true'" Include="tools/crossgen2/crossgen2/crossgen2.csproj" />
```

Comment:
Side effect of this was that we validated the project builds. The `dotnet publish` invocation in build.cmd/build.sh doesn't validate the exit code. I thought of the `dotnet publish` as a hack that will go away, but if we're making _that_ how we build the bits, it should be fixed.

Reply:
That's a good point. Long term, we want to build crossgen2 as a self-contained application, and with trimming/R2R and maybe single exe options. The IL trimming removes half of the publish size.
I'll add the error checking in the build.sh/cmd files. Out of curiosity, is there a way to publish from build.proj instead of running dotnet from the command line?

Reply:
One could always do an `<Exec>` task and run `dotnet` from there, but I'm not sure it's an improvement. We could probably call the Publish target in the project, but I haven't looked into that and whether it would be an improvement.

Reply:
Easier to check the error code of the dotnet publish commant.